src/util/Constants.js

```diff
@@ -22,6 +22,7 @@ const browser = (exports.browser = typeof window !== 'undefined');
  * @property {boolean} [fetchAllMembers=false] Whether to cache all guild members and users upon startup, as well as
  * upon joining a guild (should be avoided whenever possible)
  * @property {DisableMentionType} [disableMentions='none'] Default value for {@link MessageOptions#disableMentions}
+ * @property {MessageMentionOptions} [allowedMentions='none'] Default value for {@link MessageOptions#allowedMentions}
```

Comment:
Specify actual current default value:
```suggestion
 * @property {MessageMentionOptions} [allowedMentions={}] Default value for {@link MessageOptions#allowedMentions}
```

Reply:
Holding off on committing this pending the discussion on what the default should be

Reply:
IMO the default should just be not having the property present at all or allowing all mentions, as that's what users would expect, and that's the current behavior (no need to add a breaking change)

Reply:
Setting it to `{}` maintains the default behaviour of allowing all mentions - there functionally no difference between this and undefined currently.